### DISCUSSION TITLE
[MCS-DE-GO] Resizable value windows

### DISCUSSION THREAD

**Discussion by goergenj** [microsoft/BotFramework-Composer#5371] [General]
It should be possible to resize value windows in property settings, if entry is a sentence

**Comment by a-b-r-o-w-n** [comment]
@goergenj We are experimenting with using the new discussions feature for feature requests.

Can you provide a couple of concrete examples of properties where you wanted this feature? It would be helpful to understand if we should consider this for _all_ fields or just certain ones.

cc @sangwoohaan
